.github/workflows/firebase-qa-pull-request.yml

```diff
@@ -28,6 +28,7 @@ jobs:
           echo "//npm.pkg.github.com/:_authToken=${{ secrets.GH_TOKEN_DISCO_MIR }}" >> functions/.npmrc
```

Comment:
remove npm rc stuff from qa as well

Reply:
;) I thought I removed it from `qa` and was going to wait for build status